packit/constants.py

```diff
@@ -144,6 +144,8 @@
 COPR_SOURCE_SCRIPT = """
 #!/bin/sh
 
+git config --global user.email "user-cont-team@redhat.com"
```

Comment:
I know it's used everywhere but we should use some packit-like address for this identity... (But not need to be fixed here..;)

Reply:
Would packit@redhat.com be better?

Reply:
Yes, if it is the publicly-accessible one. (I am only worried that `user-cont-team@redhat.com` is the one associated with the FAS account. But that's not relevant here.)

Reply:
What about hello@packit.dev? (https://packit.dev/#contact)

Reply:
In the end, I think this really won't be shown anywhere, it is probably being needed only for merging (git merge) with target branch for PR locally, so I think we can leave the user-cont-team@redhat.com there.

Reply:
Isn't it shown in the spec-file changelog?

Reply:
oh, yes :D ok, so I checked what do we set when we built SRPM on our own: Packit Service, user-cont-team+packit-service@redhat.com (this is also used in proposing downstream changes)...so should we unify it?